drivers/vhm/vhm_ioreq.c

```diff
@@ -149,6 +149,7 @@ static int alloc_client(void)
 	if (!client)
 		return -ENOMEM;
 	client->id = i;
+	client->kthread_exit = true;
```

Comment:
kthread_exit is used to exit the kthread loop.
The gvt-g doesn't use kthread mode. Can you add the client->vhm_create_kthread to check the below loop in the function of acrn_ioreq_destroy_client_pervm?
       >        while (!client->kthread_exit)
                msleep(10);